scripts/geome_things.py

```diff
@@ -19,23 +19,9 @@
 CONCURRENT_DOWNLOADS = 10
```

Comment:
Cleanup -- this was representative of the changes for all providers so you'll see the same lines deleted later on in the PR.